File: Python Word examples/documentProperties/deleteCustomDocumentProperties.py
import spirecloudword
from spirecloudword.configuration import Configuration as WordConfiguration

appId = "your id"
appKey = "your key"

configuration = WordConfiguration(appId, appKey)
api = spirecloudword.api.document_properties_api.DocumentPropertiesApi(configuration)

#Support file format: docx, doc, docm, dot, dotx, dotm, rtf
file_name = "DocumentProperties.docx"
custom_property_name_1 = 'E-iceblue'
extra = {'folder': 'input',
         'dest_file_path': '/output/DeleteCustomDocumentProperties.docx'}
api.delete_custom_document_properties("DocumentProperties.docx", custom_property_name_1, **extra)

File: PHP Word Examples/DocumentPropertiesApiDemo/deleteCustomDocumentProperties.php
<?php

use Spire\Cloud\Word\Sdk\Client\Api\DocumentPropertiesApi;
use Spire\Cloud\Word\Sdk\Client\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl="https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey,$baseUrl);
$apiInstance = new DocumentPropertiesApi($configuration);

$fileName = "DocumentProperties.docx";
$propertie_name_1 = "Date";
$folder = "input";
$storage = null;
$password = null;
$dest_file_path = "output/DeleteCustomDocumentProperties.docx";
$apiInstance->deleteCustomDocumentProperties($fileName, $propertie_name_1, $dest_file_path,  $password,$folder, $storage);